src/Button.story.tsx

```diff
@@ -0,0 +1,309 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import React, { ComponentProps } from "react";
+import { storiesOf } from "@storybook/react";
+import { Button } from "./Button";
+// @ts-ignore We need to use `.tsx` because this will live next to the
+// transpiled files that we can't import.
+import { IconShip2 } from "../icons/IconShip2.tsx";
+import { colors } from ".";
+import * as typography from "./typography";
+
+const iconElement = <IconShip2 css={{ width: "100%", height: "100%" }} />;
+
+const DemoSection: React.FC<{
```

Comment:
This is going to be a constant throughout the space kit story books, so many we can make it a shared RFC?

Reply:
That's a great idea. Yes!